auth-api/src/auth_api/resources/invitation.py

```diff
@@ -131,3 +132,48 @@ def delete(invitation_id):
         except BusinessException as exception:
             response, status = {'code': exception.code, 'message': exception.message}, exception.status_code
         return response, status
+
+
+@cors_preflight('GET,OPTIONS')
```

Comment:
'validate' and 'confirm' doesn't sound correct semantically, also its better to avoid verbs. 'validate' is basically a GET by token (and may be an url param to return only active). And 'confirm' is a PATCH on invitations which can return 204.

Reply:
As discussed, changed the verb in url to tokens.